tools/dotcms-cli/cli/src/main/java/com/dotcms/cli/command/files/FilesPush.java

```diff
@@ -190,6 +155,60 @@ public Integer call() throws Exception {
         return CommandLine.ExitCode.OK;
     }
 
+    private void header(int count, LocalPathStructure localPaths,
+            StringBuilder outputBuilder) {
+        outputBuilder.append(count == 0 ? "\r\n" : "\n\n").
```

Comment:
Is this going to work properly for windows ( \r\n )vs linux ( \n ). linux does not normally use the \r

Reply:
true. But I did do it. I'm gonna have to clean it up